pycheribuild/projects/cross/gdb.py

```diff
@@ -240,7 +240,10 @@ class BuildGDB(BuildGDBBase):
     repository = GitRepository(
         "https://github.com/CTSRD-CHERI/gdb.git",
         # Branch name is changed for every major GDB release:
-        default_branch="mips_cheri-8.3", force_branch=True,
+        default_branch="mips_cheri-8.3",
+        old_branches={"mips_cheri_7.12": "mips_cheri-8.3",
```

Comment:
It would be handy to use self.default_branch as the value in the dictionary perhaps.

Reply:
I guess you could assign special meaning to None as self.default_branch

Reply:
Hmm, that's an interesting idea, but that seems kind of odd?  It also makes it a bit more of a pain to deal with in the implementation.  The other option I've thought about for the gdb case is just using a local default_branch variable in this function.

Reply:
Yeah that's less gross